.NET/Microsoft.Recognizers.Definitions.Common/Portuguese/DateTimeDefinitions.cs

```diff
@@ -26,6 +26,7 @@ public static class DateTimeDefinitions
       public const string TillRegex = @"(?<till>\b(at[eé]h?|[aà]s|ao?)\b|--|-|—|——)(\s+\b(o|[aà](s)?)\b)?";
       public static readonly string RangeConnectorRegex = $@"(?<and>(e\s*(([àa]s?)|o)?)|{BaseDateTime.RangeConnectorSymbolRegex})";
       public const string DayRegex = @"(?<day>(?:3[0-1]|[1-2]\d|0?[1-9]))(?=\b|t)";
+      public const string WrittenDayRegex = @"(?<day>um|dois|tres|três|quatro|cinco|seis|sete|oito|nove|dez|onze|doze|dezena|dezenas|treze|catorze|quatorze|quinze|dezesseis|dezasseis|dezessete|dezassete|dezoito|dezenove|dezanove|vinte(\s+e\s+(um|dois|duas|tres|três|quatro|cinco|seis|sete|oito|nove))?|trinta(\s+e\s+um)?)";

```

Comment:
Less repetition than French, but this could also be optimized. Low-priority.

Reply:
modified